autoload/neomake/makers/ft/go.vim

```diff
@@ -47,3 +48,17 @@ function! neomake#makers#ft#go#govet()
             \ '%-G%.%#'
         \ }
 endfunction
+
+function! neomake#makers#ft#go#gometalinter()
+    " Only run a subset of gometalinter for speed, users can override with:
+    " let g:let g:neomake_go_gometalinter_args = ['--disable-all', ....]
+    "
+    " All linters are only warnings, the go compiler will report errors
+    return {
+        \ 'args': ['--disable-all', '--enable=gosimple', '--enable=staticcheck'],
+        \ 'append_file': 0,
+        \ 'cwd': '%:h',
+        \ 'mapexpr': 'neomake_bufdir . "/" . v:val',
+        \ 'errorformat': '%W%f:%l:%c:%m',
```

Comment:
The name "gometalinter" is kind of a misnomer, it's not really a linter, it's more of a consistent interface for quite a lot of tools, some of which happen to be linters. GMT already [classifies](https://github.com/alecthomas/gometalinter/blob/master/config.go#L134) the severity of the outputs, and the defaults are quite sensible; it also allows for user overrides of these (`--errors`, `--severity` etc). If you don't like that it reports things as errors, you can override the severities in the default args- I don't think it's possible to pick a set of default args that work for everyone, anyway- personally, I use 
```
    \     '--disable-all',
    \     '-E', 'vet',
    \     '-E', 'golint',
    \     '-E', 'errcheck',
    \     '--message-overrides', 'errcheck:{message}',
    \     '--sort', 'line',
    \     '--tests',
    \     '--exclude', 'bindata',
    \     '--exclude', '.pb.',
    \     '--enable-gc'
```
so optimizing for changing the args and capturing all information via efm is more important to me personally.